topology/Pittsburgh Supercomputing Center/PSC - Bridges/Bridges.yaml

```
@@ -1,11 +1,13 @@
+Disable: false
 Production: true
 SupportCenter: Self Supported
-GroupID: 506
-GroupDescription: This is the cluster for the Bridges HostedCE
+GroupID: 506 
```

```suggestion
GroupID: 506
```

Looks like some extra whitespace snuck in